stable/spring-cloud-data-flow/templates/server-deployment.yaml

```diff
@@ -65,7 +65,11 @@ spec:
         - name: SPRING_CLOUD_KUBERNETES_SECRETS_PATHS
           value: /etc/secrets
         - name: SPRING_CLOUD_KUBERNETES_CONFIG_NAME
+          {{- if .Values.server.defaultConfig }}
```

Comment:
`defaultConfig` and `existingConfigMap` don't exist anymore

Reply:
My bad I have totally forgot to amend the change